src/shared/components/MapTo.tsx

```diff
@@ -226,7 +226,14 @@ export const MapToDropdownBasic = ({
   children,
   config,
 }: MapToDropdownBasic) => (
-  <Dropdown visibleElement={<Button variant="tertiary">{children}</Button>}>
+  <Dropdown
+    // eslint-disable-next-line react/no-unstable-nested-components
+    visibleElement={(onClick: () => unknown) => (
```

Comment:
Franklin API change to avoid internal use of `cloneElement`, they recommended to pass the property to be added to the visible Button to be passed through function argument and rely on the consumer of the component to use it instead of it being injected when clone the component